pkg/controller/migplan/validation.go

```diff
@@ -280,6 +280,10 @@ func (r ReconcileMigPlan) validateNamespaces(plan *migapi.MigPlan) error {
 
 func (r ReconcileMigPlan) validateNamespaceLengthForDVM(plan *migapi.MigPlan) []string {
 	items := []string{}
+	// If validation for destination cluster ref failed, we can't check this
```

Comment:
This does not appear to be a log change, but rather a bugfix is that true?

Reply:
Oops I forgot to remove this here. I submitted a bugfix PR that just got merged to master separately from this PR https://github.com/konveyor/mig-controller/pull/1000